cloudinit/distros/gentoo.py

```diff
@@ -261,5 +267,13 @@ def convert_resolv_conf(settings):
             result += "nameserver %s\n" % ns
     return result
 
+def convert_resolv_conf_search(settings):
+    """Returns a settings string formatted for resolv.conf."""
+    result = ""
+    if isinstance(settings, list):
+        for domain in settings:
+            result += " %s" % domain
+    return "domain" + result + "\n"
```

Comment:
The [docs](https://wiki.gentoo.org/wiki/Resolv.conf#domain) say domain is obsolete. Think we should use `search` instead?

Reply:
Just noticed it also says "The search option accepts up to six domains". Do you know what the failure mode is? If 7 are provided is the 7th ignored? Or is the config invalid and won't work at all?

Either way we should probably warn if >6.

Reply:
To the questions, I don't know.  Yes I agree, if it will fail with more than 6, then a warning is warranted.